### PR TITLE
Create service via Connectivity Adapter 

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Create service via Connectivity Adapter 

**Related issue(s)**
#642

### CODE REVIEW

File: components/connectivity-adapter/internal/appregistry/service/converter.go [hunk 1]
```diff
@@ -11,14 +12,23 @@ func NewConverter() *converter {
 	return &converter{}
 }
 
-func (c *converter) DetailsToGraphQL(in model.ServiceDetails) graphql.Application {
-	panic("not implemented")
+func (c *converter) DetailsToGraphQLInput(in model.ServiceDetails) (graphql.ApplicationRegisterInput, error) {
+	return graphql.ApplicationRegisterInput{
```
Comment (dbadura) [inline components/connectivity-adapter/internal/appregistry/service/converter.go]: Why you don't use `in` param?

Comment (dbadura) [reply components/connectivity-adapter/internal/appregistry/service/converter.go]: It would be nice to have test for this method.

Comment (pkosiec) [reply components/connectivity-adapter/internal/appregistry/service/converter.go]: Added a comment to clarify what's going on here 🙂

File: components/connectivity-adapter/internal/appregistry/service/handler.go [hunk 2]
```diff
@@ -82,3 +165,10 @@ func (h *Handler) Delete(writer http.ResponseWriter, request *http.Request) {
 
 	writer.WriteHeader(http.StatusNoContent)
 }
+
+func (h *Handler) closeBody(rq *http.Request) {
+	err := rq.Body.Close()
+	if err != nil {
+
```
Comment (crabtree) [inline components/connectivity-adapter/internal/appregistry/service/handler.go]: It is worth to log that fact ;)

Comment (pkosiec) [reply components/connectivity-adapter/internal/appregistry/service/handler.go]: Right 😄 I missed that somehow

**Pull request merged**